Library/Homebrew/Gemfile.lock

```diff
@@ -55,7 +55,7 @@ GEM
       parallel
     parser (2.7.1.4)
       ast (~> 2.4.1)
-    patchelf (1.1.1)
+    patchelf (1.2.0)
```

Comment:
In order for this to work without refinements we need `1.2.0`.
updating  Gemfile(.lock) is a must

Reply:
This has been merged, you can rebase this.